examples/curio/curio-server.py

```diff
@@ -190,6 +191,13 @@ def __init__(self, sock, root):
 
 
 if __name__ == '__main__':
-    kernel = Kernel()
-    kernel.add_task(h2_server(('', 8080), sys.argv[1]))
+    host = sys.argv[2] if len(sys.argv) > 2 else "localhost"
+    kernel = Kernel(with_monitor=True)
+    kernel.add_task(h2_server((host, 5000),
+                              sys.argv[1],
+                              "{}.crt.pem".format(host),
+                              "{}.key".format(host)))
```

Comment:
This change adjusts the example to stop working with the cert and key that are stored in the repository. I think it'd be better if it didn't.